exonum-java-binding-core/rust/src/proxy/transaction.rs

```diff
@@ -119,3 +119,44 @@ impl Message for TransactionProxy {
         &self.raw
     }
 }
+
+/// Handles exceptions after executing transactions
+///
+/// The TransactionExecutionException and its descendants are converted into `Error`s with their
+/// descriptions. The rest (Java and JNI errors) are treated as unrecoverable and result in a panic.
+///
+/// Panics:
+/// - Panics if there is some JNI error.
+/// - If there is a pending Java throwable that IS NOT an instance of the
+/// `TransactionExecutionException`.
+fn check_transaction_execution_result<T>(
+    env: &JNIEnv,
+    result: JniResult<T>,
+) -> Result<T, ExecutionError> {
+    result.map_err(|jni_error| match jni_error.0 {
+        JniErrorKind::JavaException => {
+            let exception = get_and_clear_java_exception(env);
+            let message = unwrap_jni_verbose(env, get_exception_message(env, exception));
```

Comment:
Do we have to use `unwrap_jni_verbose` in this method (asking as it adds some noise)? 

From the spec, "unwraps … with verbose error message **if** Java exception occurred", but in these cases exceptions seem unlikely (you just call two getters and a JNI API method).